test/libsyncengine/jobs/network/testnetworkjobs.cpp

```diff
@@ -61,23 +61,30 @@ static const std::string desktopTeamTestDriveName = "Test Desktop App";
 static const std::string bigFileDirName = "big_file_dir";
 static const std::string bigFileName = "big_text_file.txt";
 
+std::string loadEnvVariable(const std::string &key) {
+    const std::string val = CommonUtility::envVarValue(key);
+    if (val.empty()) {
+        throw std::runtime_error("Environment variables " + key + " is missing!");
+    }
+    return val;
+}
+
 void TestNetworkJobs::setUp() {
     LOGW_DEBUG(Log::instance()->getLogger(), L"$$$$$ Set Up");
 
-    const std::string userIdStr = CommonUtility::envVarValue("KDRIVE_TEST_CI_USER_ID");
-    const std::string accountIdStr = CommonUtility::envVarValue("KDRIVE_TEST_CI_ACCOUNT_ID");
-    const std::string driveIdStr = CommonUtility::envVarValue("KDRIVE_TEST_CI_DRIVE_ID");
-    const std::string remoteDirIdStr = CommonUtility::envVarValue("KDRIVE_TEST_CI_REMOTE_DIR_ID");
-    const std::string apiTokenStr = CommonUtility::envVarValue("KDRIVE_TEST_CI_API_TOKEN");
-
-    if (userIdStr.empty() || accountIdStr.empty() || driveIdStr.empty() || remoteDirIdStr.empty() || apiTokenStr.empty()) {
-        throw std::runtime_error("Some environment variables are missing!");
-    }
-
+    const std::string userIdStr = loadEnvVariable("KDRIVE_TEST_CI_USER_ID");
```

Comment:
Would it be possible to reuse as much as possible of this setup code inside as a method of test class?

Reply:
Done

Reply:
I would have expected to see the whole block of 5 string creations (plus possibly some of the subsequent instructions) to be part of a re-usable method. Is it something you did but didn't push?